appbench/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
nit: Feel free to disagree, but for me "appbench" is too much of abbreviations :)
Why not simply `application-benchmarks` ?

Reply:
I'm fine with that.